internal/controllers/tenant/tenant_reconciler_function.go

```
@@ -76,20 +78,22 @@ func (b *FunctionBuilder) Build() (result *function, err error) {
 	}
 
 	result = &function{
-		logger:         b.logger,
-		tenantsClient:  privatev1.NewTenantsClient(b.connection),
-		idpManager:     b.idpManager,
-		maskCalculator: masks.NewCalculator().Build(),
+		logger:          b.logger,
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)